migrations/V105__Update_Payment_Question_Remove_Funding_Structure_Add_Trust_Fund_Types.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE public.plan_payments
+DROP COLUMN IF EXISTS funding_structure;
+
+CREATE TYPE trust_fund_type AS ENUM (
+  'MEDICARE_PART_A_HI_TRUST_FUND'
+  'MEDICARE_PART_B_SMI_TRUST_FUND'
+);
+
+ALTER TABLE public.plan_payments
+  ADD COLUMN funding_source_trust_fund_type trust_fund_type DEFAULT NULL,
+  ADD COLUMN funding_source_r_trust_fund_type trust_fund_type DEFAULT NULL;

```

Comment:
Do these need to be arrays? It looks like we should be able to select HI and/or SMI, not just one or the other.